awscli/data/workmail/2017-10-01/completions-1.json

```diff
@@ -4,7 +4,7 @@
     "Aliase": {
       "operation": "ListAliases",
       "resourceIdentifier": {
-        "Aliases": "Aliases[]"
+        "Aliase": "Aliases[]"
```

Comment:
Any ideas on how we got `Aliase` out of this?

Reply:
So the bug was that we. weren't actually using the `nltk` singularizer to generate these values.  I think there's a tradeoff either way.  The NLTK version can't handle domain specific terms (e.g `Arns-> Arn`), but our version can't handle english edge cases.  I'd prefer to just rip out the nltk stuff for now until we can get it robust enough.  It does mean that cases like the above will still happen because we can't differentiate between `Names -> Name` vs. `Aliases -> Alias`.